frontend/src/individual_edit.tsx

```diff
@@ -0,0 +1,424 @@
+                getOptionLabel={(option: OptionType) => option.label}
+                renderInput={(params) => <TextField {...params} label="Far" className={style.control} variant={inputVariant} margin="normal" />}
+                onChange={(event: any, newValue: OptionType | null) => {
+                  updateField('father', asIndividual(newValue?.value))
+                }}
+              />
+            </div>
+            <div className={style.flexRow}>
+              <Autocomplete
+                options={colorOptions}
+                value={colorOptions.find(option => option.value == individual.colour) ?? colorOptions[0]}
+                getOptionLabel={(option: OptionType) => option.label}
+                renderInput={(params) => <TextField {...params} label="Färg" className={style.control} variant={inputVariant} margin="normal" />}
+                onChange={(event: any, newValue: OptionType | null) => {
+                  updateField('colour', newValue?.value ?? '')
+                }}
+              />
+              <TextField
+                label="Färgantecking"
+                variant={inputVariant}
+                className={style.control}
+                multiline
+                rows={3}
+                value={individual.colour_note ?? ''}
+                onChange={(event) => {updateField('colour_note', event.currentTarget.value)}}
+              />
+            </div>
+            <TextField
+              label="Anteckningar"
+              variant={inputVariant}
+              className={style.wideControl}
+              multiline
+              rows={4}
+              value={individual.notes ?? ''}
+              onChange={(event) => {updateField('notes', event.currentTarget.value)}}
+            />
+          </div>
+          <div className={style.formPane}>
+            <div className={style.titleText}>
+              Mått
+            </div>
+            <h3>Vikter</h3>
+            <ul>
+              {individual.weights && individual.weights.map((w: DateWeight, i: number) =>
+                <li key={i} className={style.measureList}>
+                  {`${new Date(w.date).toLocaleDateString('sv')} - ${w.weight} kg`}
+                  <span className={style.listButton}>
+                    [<a className={style.scriptLink} onClick={() => removeMeasure('weights', i)}>
+                      Delete
+                    </a>]
+                  </span>
+                </li>)}
+            </ul>
+            <div className={style.flexRow}>
+              <TextField label='Mätningsdatum' className={style.control}
+                value={weightDate}
+                type='date'
+                variant={inputVariant}
+                InputLabelProps={{
+                  shrink: true,
+                }}
+                onChange={(e: any) => {setWeightDate(e.target.value)}}
+                />
+              <TextField label='Mätningsdatum' className={style.control}
```

Comment:
```suggestion
              <TextField label='Vikt' className={style.control}
```
Both fields have the same label should this be Vikt instead?

Reply:
yes! of course! good catch!